cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/query/BaseQuery.java

```diff
@@ -2,12 +2,17 @@
 
 import java.util.HashMap;
 
-abstract class BaseQuery extends HashMap<String, Object>
+public abstract class BaseQuery extends HashMap<String, Object>
 {
     @Override
     public Object put( String key, Object value )
     {
         if ( value == null ) return super.remove( key );
         return super.put( key, value );
     }
+
+    public enum Filter
```

Comment:
take it out so you don't have to change the visibility of BaseQuery that is useful only to those who implement Query and not who uses them